website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Upgrading to v1.4 (latest)"
+description: New features and breaking changes for dbt Core v1.4
+---
+### Resources
+
+- [Changelog](https://github.com/dbt-labs/dbt-core/blob/1.3.latest/CHANGELOG.md)
+- [CLI Installation guide](/docs/get-started/installation)
+- [Cloud upgrade guide](/docs/dbt-versions/upgrade-core-in-cloud)
+
+## Breaking changes
```

Comment:
As discussed offline: Let's retitle this from `Breaking changes` to `What to know before upgrading` — and let's do the same rename for all migration guides v1.1-v1.3